src/System.Reflection/tests/ModuleTests.cs

```diff
@@ -41,24 +42,20 @@ public static void FullyQualifiedNameTest(TypeInfo typeInfo)
         [Theory]
         [InlineData(typeof(Attr), 77, "AttrSimple")]
         [InlineData(typeof(Int32Attr), 77, "Int32AttrSimple")]
-        [InlineData(typeof(Int64Attr), (Int64)77, "Int64AttrSimple")]
+        [InlineData(typeof(Int64Attr), (long)77, "Int64AttrSimple")]
         [InlineData(typeof(StringAttr), "hello", "StringAttrSimple")]
         [InlineData(typeof(EnumAttr), PublicEnum.Case1, "EnumAttrSimple")]        
-        public static void CustomAttributesTest<CtorArg, NamedArg>(Type attrType, CtorArg expectedCtorValue, NamedArg expectedNamedValue)
+        public void CustomAttributes<CtorArg, NamedArg>(Type attrType, CtorArg expectedCtorValue, NamedArg expectedNamedValue)
         {
             Module module = typeof(ModuleTest).GetTypeInfo().Module;
-            var customAttrs = module.CustomAttributes.GetEnumerator();
 
             CustomAttributeData attribute = module.CustomAttributes.SingleOrDefault(a => a.AttributeType.Equals(attrType));
-
-            Assert.NotNull(attribute);
```

Comment:
Not sure - I assumed an NRE would be enough if this failed.

I addressed your comment by replacing `SingleOrDefault` with `Single`, so that we get an explicit error message detailing that we couldn't find the attribute mentioned